src/dbus/common/dbus_message_helper.hpp

```diff
@@ -286,6 +290,12 @@ template <> struct DBusTypeTrait<RadioCoexMetrics>
     static constexpr const char *TYPE_AS_STRING = "(uuuuuuuuuuuuuuuuuub)";
 };
 
+template <> struct DBusTypeTrait<BorderRoutingCounters>
+{
+    // struct of { uint32, uint32, uint32, uint32 }
```

Comment:
Per the code it should be `{ { uint32, uint32 }, { uint32, uint32 }, { uint32, uint32 }, { uint32, uint32 } }`.